src/OrchardCore.Modules/OrchardCore.CustomSettings/Recipes/CustomSettingsStep.cs

```diff
@@ -13,15 +12,12 @@ namespace OrchardCore.CustomSettings.Recipes
     /// </summary>
     public class CustomSettingsStep : IRecipeStepHandler
     {
-        private readonly ISiteService _siteService;
-        private readonly CustomSettingsService _customSettingsService;
+        private readonly ISiteService _siteService;        
 
         public CustomSettingsStep(
-            ISiteService siteService,
-            CustomSettingsService customSettingsService)
+            ISiteService siteService)
```

Comment:
I think we can move the param inline instead of two lines

Reply:
Constructor parameters are inline now.